dump1090.c

```diff
@@ -738,6 +744,40 @@ MODES_DUMP1090_VARIANT " " MODES_DUMP1090_VERSION
     );
 }
 
+#ifdef _WIN32
+void showCopyright(void) {
+    uint64_t llTime = time(NULL) + 1;
+
+    printf(
+        "-----------------------------------------------------------------------------\n"
+        "|                        dump1090 ModeS Receiver         Ver : " MODES_DUMP1090_VERSION " |\n"
+        "-----------------------------------------------------------------------------\n"
+        "\n"
+        " Copyright (c) 2014-2016 Oliver Jowett <oliver@mutability.co.uk>"
+        "\n"
+        " All rights reserved.\n"
+        "\n"
+        " THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS\n"
+        " ""AS IS"" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT\n"
+        " LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR\n"
+        " A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT\n"
+        " HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,\n"
+        " SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT\n"
+        " LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,\n"
+        " DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY\n"
+        " THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT\n"
+        " (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE\n"
+        " OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.\n"
+        "\n"
+        " For further details refer to <https://github.com/mutability/dump1090>\n"
+        "\n"
+    );
+
+    // delay for 1 second to give the user a chance to read the copyright
+    while (llTime >= time(NULL)) {}
+}
+#endif
+
```

Comment:
Well:

 * There's no requirement in either the BSD or GPL licenses to display this at runtime;
 * It's the wrong license text anyway since this fork is GPL;
 * There's nothing Windows-specific about this.

Blindly copying stuff from the MalcolmRobb fork is a bad idea.